docs/semgrep-ci/configuration-reference.md

```diff
@@ -31,6 +31,8 @@ since branching off from your `main` branch, set
 SEMGREP_BASELINE_REF=main
 ```
 
+It is not recommended to do diff-aware scans on any branches that are also doing full-project scans. This is because Semgrep App assumes that all previous findings have been fixed if the branch they are on is re-scanned and they do not show up.
```

Comment:
```suggestion
It is not recommended to perform diff-aware scans of branches on which Semgrep App also performs full-project scans. This is because all findings detected during the diff-aware scan will be marked as fixed if Semgrep App doesn’t detect them upon the full-project scan.
```

Trying some copy editing but not sure it’s much better. Do what you like with it!

Reply:
I didn't love this either... tried a third option 😄